### ISSUE TITLE
Overlapping buttons in Chrome

### ISSUE THREAD

**Issue by brigsz** [agrc/PEL#73]
In Chrome Version 39.0.2171.95 (64-bit)  the "requesting report" and the "download report URL" show on top of each other.

**Comment by brigsz** [comment]
![screen shot 2014-12-17 at 4 23 50 pm](https://cloud.githubusercontent.com/assets/6306463/5481039/32b69372-8609-11e4-98a5-46ada22a3b6b.png)

**Comment by steveoh** [comment]
will you try this again. I am unable to replicate. I think maybe 1cbd1f058b3e9f5fdd7d4af1ae0bf2ca0153d5da fixed this?

**Comment by steveoh** [comment]
i'm going to assume this is a race condition between [this](https://github.com/agrc/PEL/blob/master/src/app/_ReportWizardControlPanel.js#L388) and [this](https://github.com/agrc/PEL/blob/master/src/app/_ReportWizardControlPanel.js#L426) as it doesn't happen every time.

**Comment by brigsz** [comment]
I will check with the other users, I have had it happen every time I use PEL

**Comment by steveoh** [comment]
Are you double clicking? What version of chrome?

**Comment by brigsz** [comment]
I don't believe I am double clicking, I click to submit and then just wait.  At some point there is a refresh and I see the two buttons on top of each other. I am using Chrome Version 39.0.2171.95 (64-bit)
